src/main/java/core/basesyntax/exception/IllegalDateParametersException.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.exception;
+
+public class IllegalDateParametersException extends Exception {
+    public IllegalDateParametersException(String msg) {
```

Comment:
```suggestion
    public IllegalDateParametersException(String message) {
```

Reply:
please, be careful to fix all comments or ask if you have a question

Reply:
Sofia, can you explain what do you mean? 
for make constants for formatter I should create a class?